crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -884,13 +884,18 @@ def _(flag: bool):
 
 ## Objects of all types have a `__class__` method
 
+The type of `x.__class__` is the same as `x`'s meta-type. `x.__class__` is always the same value as
```

Comment:
"always" is a dangerous word 😆 pretty sure it's possible to break this invariant at runtime with a C extension type. But I also think it's reasonable for us to consider it an invariant.

Reply:
Ugh, really?? I thought that was one of the Big Things that was fixed as part of the 2-to-3 transition 😆 but you probably know better than I!

Reply:
It's possible I'm wrong and this is no longer possible for C extension types either! Not going to go down the rabbit hole of experimenting with it to find out!!